docs/skeleton.md

```diff
@@ -0,0 +1,4 @@
+
+# Skeleton
+
+Plugin documentation
```

Comment:
I would add some notes in here about how fieldmanual will look for this directory and pull in any files. Or add it to the README.

Reply:
After merging the big fieldmanual changes, would it be appropriate to add this under the Plugins section there, instead?